config/_shared.json

```diff
@@ -131,7 +131,10 @@
 	"push_notification_vapid_key": "BDQlTOrVTrxRM0i-gCsrxjxOyul281c5WVdfstXaqZqTSpjSI9M-E99CvwwJRWDkZ0goa8VSqUCgwiexxcDuXCs",
 	"enable_all_sections": true,
 	"sections": {
-		"jetpack-cloud": false
+		"jetpack-cloud": false,
+		"scan": false,
```

Comment:
do `scan` and `backups` not have the `jetpack-cloud` prefix because they don't collide with existing sections?

Reply:
Yes there is currently no scan and backup sections yet. I figured to keep things simple.